lib/models/device.coffee

```diff
@@ -1305,25 +1430,25 @@ getDeviceModel = (deps, opts) ->
 	# @public
 	# @function
 	# @memberof resin.models.device
-	#
-	# @param {String} uuid - device uuid
-	# @fulfil {String} - device statud
+	#π
```

Comment:
I... That's very weird. It appears alt-p on my machine prints pi, and I must've pressed that by accident? Now fixed.

🍰